From: uazo <uazo@users.noreply.github.com>
Date: Mon, 19 Jan 2026 12:01:31 +0000
Subject: Keep disabled CpuPerformance

Keep disabled CpuPerformance due to potential privacy issues
see https://github.com/explainers-by-googlers/cpu-performance?tab=readme-ov-file#accessibility-privacy-and-security-considerations

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../blink/renderer/platform/runtime_enabled_features.json5     | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -1273,8 +1273,7 @@
       // CPU Performance API, work in progress.
       // https://chromestatus.com/feature/5189864286978048
       // https://github.com/explainers-by-googlers/cpu-performance
-      name: "CpuPerformance",
-      status: "experimental"
+      status: "experimental", name: "CpuPerformance",
     },
     {
       name: "CrashReportingStorageAPI",
--
